driver/bpf/filler_helpers.h

```diff
@@ -1172,11 +1172,92 @@ static __always_inline int bpf_push_u32_to_ring(struct filler_data *data, u32 va
 	return PPM_SUCCESS;
 }
 
+static __always_inline int bpf_push_s32_to_ring(struct filler_data *data, s32 val)
```

Comment:
We could even try to build all of these through a macro, since they just change the `s,u{8,16,32,64}`.

Reply:
Do we have memcpy in eBPF? We could simply add a size argument and memcpy(wherever, &val, size). And then define one-line helper macros choosing the right `size` value (signed or unsigned doesn't matter here).